lib/monetary.ts

```diff
@@ -0,0 +1,174 @@
+import Big from 'big.js';
```

Comment:
I think we should use BigNumbers from ethers as we are depending on that in other places as well (see https://docs.ethers.io/v5/api/utils/bignumber/). What do you think?

Reply:
I think we will need this actually to handle decimals for the front-end, as far as I know `BigNumber` doesn't work for this.

Reply:
Sticking with big.js for now to support decimals.